doc/tutorials/using_industry_scenario_on_nuc.rst

```diff
@@ -0,0 +1,105 @@
+.. _using_industry_scenario_on_nuc:
+
+Launch Two Guest VMs on NUC Using Industry Scenario
+###################################################
+
+ACRN hypervisor supports new industry scenario
+since the tag ``acrn-2019w18.7-140000p``. In industry scenario,
+up to three Guest VMs (e.g. Vxworks + RTVM + WaaG) can be launched from
+the Service VM. This tutorial provides step by step instructions on how
+to enable the ACRN hypervisor industry scenario on Intel NUC to activate
+two post-launched VMs running Linux based guest OSes. The same process
+can be applied to launch the third VM as well.
+
+ACRN Service VM Setup
+*********************
+
+Refer to the steps in :ref:`getting-started-apl-nuc` to set up ACRN on
+Intel NUC. The target device must be capable of launching a Clear Linux Guest VM
+as a starting point.
+
+Re-build ACRN UEFI Executable
+*****************************
+
+The ACRN pre-built UEFI executable ``acrn.efi`` is compiled for SDC scenario
+by default, in which only one post-launched VM is supported.
+To activate more than one post-launched VM, you need to enable the industry
+scenario and rebuild the UEFI executable with the following steps:
+
+#. Refer to :ref:`getting-started-building` to set up the development environment
+   for re-compiling the UEFI executable from ACRN source tree.
+
+#. Enter the ``hypervisor`` directory under the ACRN source tree to
+   reconfigure the ACRN hypervisor for industry scenario.Please refer following
```

Comment:
Miss one space between "scenario." and "Please".

Reply:
Fixed.